node/middlewares/viewerCountry.ts

```diff
@@ -0,0 +1,15 @@
+import { NotFoundError } from '@vtex/api'
+
+export async function viewerCountry(ctx: Context) {
+  const {
+    request: { header },
+  } = ctx
+
+  try {
+    const countryCode = header['cloudfront-viewer-country']
+
+    ctx.body = countryCode
+  } catch {
```

Comment:
I don't think this will ever catch anything. There is nothing that can go wrong in the try block. I would remove it (and the error handler) or make a check if `coutryCode` is empty throw an error.

Reply:
You're right!